### DISCUSSION TITLE
Data requirements

### DISCUSSION THREAD

**Discussion by sunsonjay** [DICOMWG26AnnotationAHG/hackathon#6] [Q&A]
Hi, 
Would the data be provided? Or to be accessed via DICOMweb but which set to query for use in the hackathon?

Data
DICOM Slide Microscopy (SM) images
DICOM Segmentation (SEG) images
DICOM Structured Report (SR) documents

**Comment by hackermd** [comment]
Thanks for your interest @sunsonjay. I've just updated the relevant section in the README, which now provides links to documents describing the available data sets and the locations of the origin servers:
https://github.com/DICOMWG26AnnotationAHG/hackathon/tree/deda537df15c7913c9d77c5bbd1688caa1eae3fc#what-will-be-provided
